recipe/meta.yaml

```diff
@@ -221,7 +221,7 @@ outputs:
         - ld_impl_{{ target_platform }}  # [linux]
       run:
         - ld_impl_{{ target_platform }}  # [linux]
-        - tzdata
+        - python-tzdata
```

Comment:
I thought that the idea was to avoid the python package and force Python to find the `tzdata` noarch generic, no?